### ISSUE TITLE
How can I access the video duration from the API?

### ISSUE THREAD

**Issue by clemgrim** [cloudflare/stream-react#58]
So I'm using the Stream component in a straightforward way, here's a simplified version:

```tsx
export default function ({ videoId }) {
  const streamRef = useRef<StreamPlayerApi>();
  const [volume, setVolume] = useState(100);

  return (
    <Stream
            src={videoId}
            streamRef={streamRef}
            currentTime={currentTime}
            volume={volume / 100}
            onLoadedData={(e) => console.log("onLoadedData", e)}
            onDurationChange={(e) => console.log("onDurationChange", e)}
            onLoadedMetaData={(e) => console.log("onLoadedMetaData", e)}
        />
  );
}
```

So I would like to have a way to access the duration of the video, either from the  `StreamPlayerApi`, as for the volume, or as props from the `LoadedMetaData` or `DurationChange` events.
Is it possible?

**Comment by third774** [comment]
Hey! Thanks for asking! It looks like the `duration` property is missing from the `StreamPlayerApi` type information, but it is there at runtime.

The value stored in `streamRef` is the [instantiated player SDK](https://developers.cloudflare.com/stream/viewing-videos/using-the-player-api/).

[Here's a sandbox](https://codesandbox.io/s/quirky-jang-5hl0bi?file=/src/App.tsx) that shows how to access `duration` for now, but we'll get that added to the type information and double check to make sure we haven't missed any others. Thanks for bringing this up!

**Comment by clemgrim** [comment]
Amazing thank you! sorry I didn't think about logging the playerApi, I was relying 100% on the types haha. I will try that

**Comment by third774** [comment]
Gonna leave this open until it's fixed!

**Comment by third774** [comment]
This is fixed in [v1.7.0](https://github.com/cloudflare/stream-react/releases/tag/v1.7.0).
